webpack.config.js

```diff
@@ -1,6 +1,13 @@
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
 
 module.exports = {
+  entry: "./src/index.js",
+  output: {
+    path: path.resolve(__dirname, "./dist"),
+    filename: "index_bundle.js"
```

Comment:
Need to consider the cache issue for production

Reply:
`filename: "[name].[contenthash].js"` can fix it.